tests/functional/tests/osd/test_osds.py

```diff
@@ -0,0 +1,23 @@
+
+class TestOSDs(object):
+
+    def test_ceph_osd_package_is_installed(self, CephNode, Package):
+        assert Package("ceph-osd").is_installed
+
+    def test_osd_listens_on_6800(self, CephNode, Socket):
+        assert Socket("tcp://%s:6800" % CephNode["address"]).is_listening
```

Comment:
I lifted this from https://github.com/redhat-cip/openstack-serverspec/blob/master/spec/tests/rbdosd/rbdosd_spec.rb#L9

Is there a range of ports I should be checking?